benchmarks/benchmark_serving.py

```diff
@@ -60,6 +60,7 @@ class BenchmarkMetrics:
     total_input: int
     total_output: int
     request_throughput: float
+    request_goodput: float
```

Comment:
I would suggest to rename `request_goodput` to `request_qualified_throughput`. Perhaps it's a bit more clear.

Reply:
Thanks for this advice.😄 I understand your idea on code readability, but **Goodput** is the important metric name what we are introducing to the whole LLM serving and inference system community. For more context, you can refer to the [RFC](https://github.com/vllm-project/vllm/issues/8782) and [our Blog on goodput](https://hao-ai-lab.github.io/blogs/distserve/)

So I would remain as it is.😄